package-lock.json

```diff
@@ -5712,7 +5731,7 @@
         },
         "md5.js": {
             "version": "1.3.4",
-            "resolved": "https://registry.npmjs.org/md5.js/-/md5.js-1.3.4.tgz",
+            "resolved": "http://registry.npmjs.org/md5.js/-/md5.js-1.3.4.tgz",
```

Comment:
Can you do all that stuff to clean out your npm cache? So we don't get these http changes?